grammars/README.md

```diff
@@ -7,6 +7,8 @@ supports. Please raise any shortcomings in a GitHub issue.
 The definitions are somewhat scattered around the codebase; this page serves as
 an index.
 
+- [Ace](https://ace.c9.io/) — supported. The grammar is upstream.
```

Comment:
Could we add a link to the grammar in the upstream repo?

Reply:
Okay, I added a link to one of the files (the interesting one). The other one ([prql.js](https://github.com/ajaxorg/ace/blob/master/src/mode/prql.js)) is less interesting, and the third one ([prql.prql](https://github.com/ajaxorg/ace/blob/master/demo/kitchen-sink/docs/prql.prql)) is just the example for the demo.